src/ingest_validation_tools/schema_loader.py

```diff
@@ -46,9 +46,9 @@ class SchemaVersion:
     version: str = ""
     directory_path: Optional[Path] = None
     table_schema: Optional[str] = ""  # legacy only
```

Comment:
Reformatted how antibodies/contributors references are tracked (list of paths -> list of schemas)